sentences/lv/homeassistant_HassRespond.yaml

```
@@ -0,0 +1,26 @@
+language: "lv"
+intents:
+  HassRespond:
+    data:
+      - sentences:
+          - "(pasaki|saki) {response}"
+
+      - sentences:
+          - "(sveiks|sveiki|hi|čau) [home assistant]"
```

```suggestion
          - "(sveik(s|i)|čau) [home assistant]"
```
I don't think we should include `hi` in Latvian language.